examples/dut_communication_using_semi_device_control_nidigital.py

```diff
@@ -44,18 +44,20 @@
 
     # Attach to the existing 657x gRPC session.
     # Get the resource name of the instrument session for NI 657x device.
-    resource_name = semi_device_control.get_instrument_session("NI 657x").ResourceName
+    grpc_session_options = semi_device_control.get_grpc_session_options("NI 657x")
     
     # Create an insecure gRPC channel to connect to the device server
-    device_server_channel = grpc.insecure_channel("localhost:31763")
+    device_server_channel = grpc.insecure_channel(grpc_session_options.Address + ":" + str(grpc_session_options.Port))
```

Comment:
Should user be constructing this channel? Can we not create and give?

Reply:
Updated